lib/IncomingStreamTrackBridge.js

```diff
@@ -159,6 +162,7 @@ class IncomingStreamTrackBridge extends Emitter
 			//Create new stats
 			this.stats = /** @type {TrackStats} */ ({});
 		
+		const rtt = this.stream.getStream() ? this.stream.getStream().stream.GetRTT() : 0;
```

Comment:
```suggestion
		const rtt = this.stream.GetRTT();
```

it is fine to throw an exception if `this.stream` is null